src/Filter/SkipExistingProducts.php

```diff
@@ -2,16 +2,23 @@
 
 namespace Jh\Import\Filter;
 
+use Jh\Import\Config;
 use Jh\Import\Import\Record;
+use Jh\Import\Import\RequiresPreparation;
 use Magento\Framework\App\ResourceConnection;
 
-class SkipExistingProducts
+class SkipExistingProducts implements RequiresPreparation
 {
     /**
      * @var array
      */
     private $existingSkus = [];
 
+    /**
+     * @var string
+     */
+    private $skuField = 'sku';
```

Comment:
do we need default value if we always override it in prepare()?

Reply:
I had just based this PR on the file `src/Filter/SkipNonExistingProducts.php` for consistency, but I agree, probably not.

Reply:
That module is not the best quality on all areas - development start long time ago. That's why I think in new PRs we don't have to mimic some concepts, but we should aim for best quality. If there is no case in which that default value is not overriden then remove it please.

Reply:
also that property should be named `$idField` instead of `$skuField`

Reply:
@bartoszkubicki please review again.